bench/lcrypto-j2me-152/LICENSE.html

```diff
@@ -0,0 +1,22 @@
+<html>
```

Comment:
Nit: could you convert this to text and rename it to LICENSE? That'll keep it consistent with all the other LICENSE files in the repo.